app/controllers/works_controller.rb

```diff
@@ -0,0 +1,88 @@
+class WorksController < ApplicationController
+  before_action :find_work, only: [:show, :edit, :update, :destroy]
+  before_action :find_work_votes, only: :show
+
+  def home
+    @works = Work.all
+    @all_works = Work.list_all_works
+    @top_ten = Work.top_ten(@all_works)
+    @spotlight = Work.spotlight(@top_ten)
+  end
+
+  def index
+    @works = Work.all
+    @all_works = Work.list_all_works
+  end
+
+  def new
+    @work = Work.new()
+  end
+
+  def show; end
+
+  def create
+    @work = Work.new(work_params)
+
+    if @work.save
+      flash[:success] = "Successfully created new #{@work.category} \"#{@work.title}!\""
+      redirect_to works_path
+    else
+      flash[:error] = "Work not created!"
+      @work.errors.messages.each do |field, messages|
+        flash[field] = messages
+      end
+      redirect_to new_work_path
+    end
```

Comment:
You should instead render the new view.

```suggestion
      flash.now[:error] = "Work not created!"
      @work.errors.messages.each do |field, messages|
        flash.now[field] = messages
      end
      render :new, status: :bad_request
    end
```